mono/metadata/w32socket-unix.c

```diff
@@ -42,9 +42,7 @@
 #ifdef HAVE_SYS_SOCKIO_H
 #include <sys/sockio.h>    /* defines SIOCATMARK */
 #endif
-#ifndef HAVE_MSG_NOSIGNAL
```

Comment:
*Arguably* the include can be skipped, as sometimes not needed, wasteful to include. Not because the file isn't there and functional. Or maybe that is too subtle to care about, and reduce config sensitivity.
Odd that the ifdef isn't otherwise used either. Perhaps, really, there is no alternate path after all.